pyspark_ai/spark_sql_chain.py

```diff
@@ -61,7 +66,7 @@ def _generate_code_with_retries(
             if self.logger is not None:
                 self.logger.info("Retrying with " + str(retries) + " retries left")
 
-            messages.append(response)
+            # messages.append(response)
```

Comment:
Please leave some message to tell reader why it should be commented.

Reply:
In this situation, when the retry happens, it appends the response back to the original prompt, making it excessively long. This can cause the model to return incorrect results again. Therefore, it's better not to include it.

Reply:
Please add comment in the code to clarify this. Thanks!